src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/ClaimController.java

```diff
@@ -57,6 +60,19 @@ public Claim getByExternalId(@PathVariable("externalId") final String externalId
         return claimService.getClaimByExternalId(externalId);
     }
 
+    @GetMapping("/{claimReference:" + CLAIM_REFERENCE_PATTERN + "}")
+    @ApiOperation("Fetch claim for given claim reference")
+    public Claim getByClaimReference(@PathVariable("claimReference") final String claimReference) {
+        return claimService.getClaimByReference(claimReference)
+            .orElseThrow(() -> new NotFoundException("Claim not found by claim reference " + claimReference));
+    }
+
+    @GetMapping("/representative/{externalReference}")
+    @ApiOperation("Fetch user claims for given external reference number")
+    public List<Claim> getClaimByExternalReference(@PathVariable("externalReference") final String externalReference) {
```

Comment:
is `externalReference` the same as `externalId`?

Reply:
ok, I can see it is. Do we want to rename all `externalId` to `externalReference` now?

Reply:
no, externalID and external reference are different things.
external reference is a user entered ID by the solicitor